rules/cross-platform/guided_onboarding_sample_rule.toml

```diff
@@ -15,18 +15,8 @@ enabled = false
 false_positives = [
     "This rule is not looking for threat activity. Disable the rule if you're already familiar with alerts.",
 ]
-from = "now-24h"
-index = [
-    "apm-*-transaction*",
-    "auditbeat-*",
-    "endgame-*",
-    "filebeat-*",
-    "logs-*",
-    "packetbeat-*",
-    "traces-apm*",
-    "winlogbeat-*",
-    "-*elastic-cloud-logs-*",
-]
+from = "now-1h"
```

Comment:
@approksiu does 1 hour make sense for this, or can we reduce it more to maybe last 30 mins? If this is specifically for guided onboarding I would assume 30m would be sufficient for data ingestion when onboarding?

Reply:
Done